src/aiida_quantumespresso/workflows/functions/get_xspectra_structures.py

```diff
@@ -63,10 +225,14 @@ def get_xspectra_structures(structure, **kwargs):  # pylint: disable=too-many-st
               all generated structure and associated symmetry data
     """
 
-    is_molecule_input = False
     elements_present = [kind.symbol for kind in structure.kinds]
     unwrapped_kwargs = {key: node for key, node in kwargs.items() if isinstance(node, orm.Data)}
-    if 'abs_atom_marker' in unwrapped_kwargs.keys():
+
+    is_molecule_input = False
+    if 'is_molecule_input' in unwrapped_kwargs:
+        is_molecule_input = kwargs['is_molecule_input'].value
```

Comment:
```suggestion
    is_molecule_input = kwargs.get('is_molecule_input', False)
```
this should work because one can use AiiDA Bool data in the `if` condition directly.